package.json

```diff
@@ -35,15 +35,18 @@
   "homepage": "https://github.com/na2hiro/Shogi.js#readme",
   "devDependencies": {
     "@types/jest": "^22.2.0",
+    "@typescript-eslint/eslint-plugin": "^5.8.1",
+    "@typescript-eslint/parser": "^5.8.1",
     "browserify": "^16.5.0",
     "clean-webpack-plugin": "^0.1.19",
+    "eslint": "^8.6.0",
     "jest": "^26.1.0",
     "pre-push": "^0.1.1",
     "ts-jest": "^26.4.3",
     "ts-loader": "^4.3.1",
     "tslint": "^5.9.1",
```

Comment:
Thanks for the pull request! It looks good.

Could you run `npm uninstall tslint` to remove the dependency?

Reply:
Thank you for your checking. I'll do it this evening.